Makefile

```diff
@@ -46,6 +46,13 @@ OLM_MAJOR_SHLIB_EXT := $(SHLIB_EXT).$(SOMAJOR)
 endif
 endif
 
+# OpenBSD doesn't support cp -a, so we'll use the equivalent
+ifeq ($(OS),OpenBSD)
+CP_A := cp -RpP
+else
+CP_A := cp -a
+endif
```

Comment:
I actually think `cp -RpP` will work everywhere; so we can probably just use that.